src/backend/api/Fusion.Resources.Api/Controllers/Requests/ApiModels/ApiResourceAllocationRequest.cs

```diff
@@ -99,7 +99,7 @@ public ApiResourceAllocationRequest(QueryResourceAllocationRequest query)
         public bool IsDraft { get; set; }
         public ApiProvisioningStatus ProvisioningStatus { get; set; }
 
-        public bool ShouldHideProposalsForProject
+        internal bool ShouldHideProposalsForProject
```

Comment:
decided to hide this property since it is not used by front-end

Reply:
A more explicit way to hide it would be to use JsonIgnore on it..